matchzoo/tasks/classification.py

```diff
@@ -0,0 +1,34 @@
+"""Classification task."""
+
+import keras
+
+from matchzoo import engine
+
+
+class Classification(engine.BaseTask):
+    """Classification task."""
+
+    def __init__(self, num_classes: int = 2):
+        super().__init__()
+        if not isinstance(num_classes, int):
+            raise TypeError
```

Comment:
```python
import this

Explicit is better than implicit.
```

Reply:
I agree, and please be explicit about your point. If you mean `num_classes` shouldn't have a default value, it's a valid point. I thought about that, but since binary classification is the most used classification type, I think it's ok to assume it's the `default`.

Reply:
raise TypeError(what happened?)